VORON-0/Firmware/printer.cfg

```diff
@@ -0,0 +1,239 @@
+# This file contains common pin mappings for the BIGTREETECH SKR
+# MINI. To use this config, the firmware should be compiled for the
+# STM32F103 with a "28KiB bootloader".
+
+# The "make flash" command does not work on the SKR mini. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "firmware.bin" on an SD card and then restart the SKR
+# mini with that SD card.
+
+# See the example.cfg file for a description of available parameters.
+
+[mcu]
+serial: /dev/serial/by-id/usb-Klipper_stm32f103xe_37FFD8054246383440750857-if00
+
+[stepper_x]
+step_pin: PB13
+dir_pin: PB12
+enable_pin: !PB14
+step_distance: .0125
+endstop_pin: PC0 
+position_endstop: 120
+position_max: 120
+homing_speed: 20   #Max 100
+homing_retract_dist: 5
+homing_positive_dir: true
+
+[tmc2209 stepper_x]
+uart_pin: PB15
+microsteps: 16
+interpolate: True
+run_current: .25
+hold_current: .25
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+[stepper_y]
+step_pin: PB10
+dir_pin: PB2
+enable_pin: !PB11
+step_distance: .0125
+endstop_pin: PC1
+position_endstop: 120
+position_max: 120
+homing_speed: 20   #Max 100
+homing_retract_dist: 5
+homing_positive_dir: true
+
+[tmc2209 stepper_y]
+uart_pin: PC6
+microsteps: 16
+interpolate: True
+run_current: .25
+hold_current: .25
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+[stepper_z]
+step_pin: PB0
+dir_pin: PC5
+enable_pin: !PB1
+step_distance: .000625
+endstop_pin: PC2
+#position_endstop: -.10
+position_max: 120
+position_min: -1
+homing_speed: 10   #Max 100
+second_homing_speed: 3.0
+homing_retract_dist: 3.0
+
+[tmc2209 stepper_z]
+uart_pin: PC10
+microsteps: 16
+interpolate: True
+run_current: .8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+
+[extruder]
+step_pin: PB3
+dir_pin: PB4
+enable_pin: !PD2
+step_distance: .0024
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC8
+sensor_type: NTC 100K 3950
+#EPCOS 100K B57560G104F
+sensor_pin: PA0
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 270
+min_extrude_temp: 170
+max_extrude_only_distance: 780.0
+max_extrude_cross_section: .8
+Pressure_advance: 0.0
+pressure_advance_smooth_time: 0.040
+
+[tmc2209 extruder]
+uart_pin: PC11
+microsteps: 16
+interpolate: True
+run_current: 0.2
+hold_current: 0.2
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+[heater_bed]
+heater_pin: !PC12
+sensor_type: NTC 100K 3950
+#sensor_type: NTC 100K MGB18-104F39050L32
```

Comment:
uncomment this line

Reply:
are you sure? the keenovo site specifically calls out the "NTC 100K thermistor ( Beta 25/50 3950K-1%, Click Here for the R-T datasheet)"

Reply:
good call. You're right, he majority of us are using Keenovo.